sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/README.md

```diff
@@ -217,6 +217,39 @@ public static async Task Run(
 }
 ```
 
+### Configuring the ServiceBusClient
+
+The `ServiceBusClient` can be configured using the  [settings](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-service-bus#additional-settings-for-version-5x) available in `host.json` for the majority of scenarios.  Some less common scenarios, however, require configuring options not available in the settings.  For example, when hosting your Function locally, it is sometimes necessary to specify the web sockets transport to request that the client use port 443 rather than the default 5671 and 5672 which are blocked by some networks.
```

Comment:
the proxy transport is available in the settings that can be specified in host.json.